src/locales/en/translation.json

```diff
@@ -95,9 +95,10 @@
     "undefined": "Undefined",
     "unknown": "Unknown",
     "value": "Value",
-    "valueInToken": "{{value}} {{ticker}}",
-    "roundedValueInToken": "{{value}}… <TickerLink />",
-    "valueInTokenWithLink": "{{value}} <TickerLink />",
+    "valueLong": "{{value, number}}",
+    "valueInToken": "{{value, number}} {{ticker}}",
+    "roundedValueInToken": "{{value, number}}… <TickerLink />",
+    "valueInTokenWithLink": "{{value, number}} <TickerLink />",
```

Comment:
I don't know how to better name these and lessThanAmount and valuePair. Future issue?